index.html

```diff
@@ -17,128 +18,277 @@
+               <div class="canopy"></div>
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <div class="lang-btn">
+                  <ul>
+                     <li><a href="#"><button>html</button></a></li>
+                     <li><a href="#"><button>css</button></a></li>
+                     <li><a href="#"><button>javaScript</button></a></li>
+                  </ul>
+               </div>
+               <div class="see-project-btn"><button>See Project</button></div>
+         </section>
+         <section class="card-2">
+               <div class="card-2-bg"></div>
+               <h2>Tonic</h2>
+               <div class="canopy"></div>
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <div class="lang-btn">
+                  <ul>
+                     <li><a href="#"><button>html</button></a></li>
+                     <li><a href="#"><button>css</button></a></li>
+                     <li><a href="#"><button>javaScript</button></a></li>
+                  </ul>
+               </div>
+               <div class="see-project-btn"><button>See Project</button></div>
+         </section>
+         <section class="card-3">
+               <div class="card-3-bg"></div>
+               <h2>Tonic</h2>
+               <div class="canopy"></div>
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <div class="lang-btn">
+                  <ul>
+                     <li><a href="#"><button>html</button></a></li>
+                     <li><a href="#"><button>css</button></a></li>
+                     <li><a href="#"><button>javaScript</button></a></li>
+                  </ul>
+               </div>
+               <div class="see-project-btn"><button>See Project</button></div>
+         </section>
+         <section class="card-4">
+               <div class="card-4-bg"></div>
+               <h2>Tonic</h2>
+               <div class="canopy"></div>
+               <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+               <div class="lang-btn">
+                  <ul>
+                     <li><a href="#"><button>html</button></a></li>
+                     <li><a href="#"><button>css</button></a></li>
+                     <li><a href="#"><button>javaScript</button></a></li>
+                  </ul>
+               </div>
+               <div class="see-project-btn"><button>See Project</button></div>
+         </section>
+      </section>
+      <section class="ams">
+         <h2>About Myself</h2>
+         <p>
+            Hello I'm a software developer! I can help you build a product, feature or website
+            Look through some of my work and experiance! If you like what you see and have a project you need coded,
+            don't hestiate to contact me.
          </p>
```

Comment:
- [ ] **[OPTIONAL]** Kindly, consider adding some margin to the about section (on screen size <768px) to make the design consistent with the figma design. This will make your project prettier and make the user know your a awesome developper as you can precisly follow a professional web developer design 😃 💪 
**Your project design**
![image](https://user-images.githubusercontent.com/74411135/229288048-50db45eb-f354-4b1d-992b-cdf37077323c.png)